src/desktop/components/main_layout/header/view.coffee

```diff
@@ -148,24 +127,6 @@ module.exports = class HeaderView extends Backbone.View
       error: (xhr, status, errorMessage) ->
         new FlashMessage message: errorMessage
 
-  checkForPersonalizeFlash: ->
-    # Sometime '/personalize/' can exist as a redirect parameter in the URL.
-    # This causes the flash message to display at unexpected times.
-    # This ensures we check for personalize in the pathname.
-    if document.referrer.split('?')[0].match '^/personalize.*'
-      new FlashMessage message: 'Thank you for personalizing your profile'
-
-  checkForAfterSignUpAction: ->
-    afterSignUpAction = Cookies.get 'afterSignUpAction'
-    if afterSignUpAction
-      return unless @currentUser
-      { action, objectId, kind } = JSON.parse(afterSignUpAction)
-
-      ops = operations[action]
-      ops and ops(@currentUser, objectId, kind)
-
-      Cookies.expire 'afterSignUpAction'
-
```

Comment:
The previous items have been moved out of the header as they are more general / global and related to the page shell. This will help us keep track of what to keep track of when the new shell is developed.